src/PrestaShopBundle/Form/Admin/Sell/Product/Description/DescriptionType.php

```diff
@@ -168,7 +168,7 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver
             ->setDefaults([
                 'required' => false,
-                'label' => false,
+                'label' => $this->trans('Description', 'Admin.Catalog.Feature'),
```

Comment:
Could we use the existing string in Admin.Global?

Reply:
I don't think it's the same namespace 🤔 We shouldn't confuse the generic wording `Description` from the specific tab name in the product page Using two namespaces allows to customize the tab name here for the product page without impacting the generic wording `Description` which may be used in many parts of the BO

Reply:
Oh and by the way for product's description field we do use the global namespace https://github.com/PrestaShop/PrestaShop/blob/81744c8c1c4ea92d5e318d38ffa504c5d0ebe773/src/PrestaShopBundle/Form/Admin/Sell/Product/Description/DescriptionType.php#L118

In this case it doesn't match the generic use case of a description, but I still think the tab names are more specific

Reply:
You're right! If it's the name of the tab, then let's keep Admin.Catalog.Feature.